common-ui/src/main/java/com/duckduckgo/mobile/android/ui/viewbinding/FragmentViewBindingDelegate.kt

```diff
@@ -45,10 +46,14 @@ class FragmentViewBindingDelegate<T : ViewBinding>(
     init {
         fragment.viewLifecycleOwnerLiveData.observe(fragment) { lifecycleOwner ->
             lifecycleOwner.lifecycle.addObserver(
-                object : LifecycleObserver {
-                    @OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)
-                    fun onDestroy() {
+                object : DefaultLifecycleObserver {
+                    override fun onCreate(owner: LifecycleOwner) {
+                        Timber.d("Registering fragment lifecycle observer: ${fragment.javaClass.canonicalName}")
```

Comment:
I don't think we need to override onCreate just to log something right? I think we can just stick to overriding onDestroy. Wdyt?

Reply:
Yeah, it is a bit confusing just override onCreate just for logging. I did it in the first place to test lifecycle was registered in the fragment and then, I decided to leave it for consistency when we register application lifecycle. But probably not worth it just to override for this.
Now that it' tested, I will remove the log 👍